code/modules/antagonists/ert/ert.dm

```diff
@@ -225,16 +225,17 @@
 	..()
 
 /datum/antagonist/ert/families/greet()
-	to_chat(owner, "<B><font size=3 color=red>You are the [name].</font></B>")
-	to_chat(owner, "<B><font size=3 color=red>You are NOT a Nanotrasen Employee. You work for the local government.</font></B>")
-
+	to_chat(owner, "<B><font size=6 color=red>You are the [name].</font></B>")
+	to_chat(owner, "<B><font size=5 color=red>You are NOT a Nanotrasen Employee. You work for the local government.</font></B>")
+	to_chat(owner, "<B><font size=5 color=red>You are NOT a deathsquad. You are here to help innocents escape violence, criminal activity, and other dangerous things.</font></B>")
 	var/missiondesc = "After an uptick in gang violence on [station_name()], you are responding to emergency calls from the station for immediate SSC Police assistance!\n"
 	missiondesc += "<BR><B>Your Mission</B>:"
-	missiondesc += "<BR> <B>1.</B> Secure the situation and crack down on any gang activity. You can view gangsters with your sunglasses."
-	missiondesc += "<BR> <B>2.</B> There is an undercover police officer on station. Secure him, receive his intel, and extract him safely."
-	missiondesc += "<BR> <B>3.</B> Minimize civilian casualties, but defend yourself and civilians from hostile gangsters."
-	missiondesc += "<BR> <B>3.</B> If Security is found to be violating the rights of citizens, detain them as per your authority as Spinward Stellar Coalition officers."
-	missiondesc += "<BR> <B>4.</B> If the situation demands it, evacuate the station. Otherwise, remain on station and keep the peace."
+	missiondesc += "<BR> <B>1.</B> Serve the public trust."
+	missiondesc += "<BR> <B>2.</B> Protect the innocent."
+	missiondesc += "<BR> <B>3.</B> Uphold the law."
+	missiondesc += "<BR> <B>4.</B> Find the Undercover Cops."
```

Comment:
Last I checked security doesn't glow. Can you fix that?

Reply:
I don't want a cop to ignore a changeling eating an SSC citizen because they didn't see it mentioned in their objectives.

Reply:
also, i'd imagine Uphold The Law will cover that